hack/version2.py

```
@@ -0,0 +1,75 @@
+"""
+
+import git
+import os
+import subprocess
+
+
+def _get_master_branch_prefix():
+    """Extract MASTER_BRANCH_PREFIX from version2.sh."""
+    script_path = os.path.join(os.path.dirname(__file__), "version2.sh")
+
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)